Website/views/partials/toggleSecurity.ejs

```diff
@@ -1,9 +1,23 @@
+
+<script type="text/javascript">
+$(function () {
+  $('[data-toggle="tooltip"]').tooltip({
+    /* script handling explained here: https://stackoverflow.com/a/1659762 */
+    template: '<div class="tooltip tooltip-custom"><div class="tooltip-arrow"></div><script>alert(\'This is a component with a known vulnerability!\')</scr'+'ipt><div class="tooltip-inner"></div></div>'
+  })
+})
+</script>
+
+
+
 <p style="text-align:left;">
     <form method="post" action="/api/togglesecurity" style="float:right;">
-        <input type="submit" value="Toggle Security Mode">
+        <input class="btn btn-secondary" type="submit" value="Toggle Security Mode">
     </form>  
+
+
     <% if ( locals.uname ) { %>
-        Logged in as <%= locals.uname %>
+        <button type="button" class="btn btn-secondary" data-toggle="tooltip" data-placement="bottom" title="<%= locals.uname %>">Logged in</button>
```

Comment:
I think if we are going to give the login sign a button look we should also allow it to /logout on click

Reply:
Hmm, the only reason it has a button look is to communicate an action, and the action is the tooltip. Similar to how sometimes tooltips will be styled similar to hyperlinks but they aren't necessarily always be a hyperlink. If it was plaintext no one would know there was an action. Can you think of another way to style it so that it communicates the tooltip but won't leave people expecting a button?

Since it is so obtrusive on the insecure site I don't want it to be the logout button though, as it makes logging out prohibitively difficult.

Reply:
Well we could leave the regular logout on the navbar and just have this button logout as an extra. I think it *looks* fine it's just that the button like look implies that is should be used for something. I can't think of a better styling atm though

Reply:
We could go with something like [this](https://getbootstrap.com/docs/4.5/components/tooltips/#disabled-elements). If we disable the buttonlike highlight i think that would be enough

Reply:
Yeah, at a minimum I think changing the color would be better since the other button is an actual link that does something, so differentiating it would be good. I'll explore that tomorrow.

Reply:
How's this? Pic from latest commit:
<img width="562" alt="Screen Shot 2020-11-21 at 3 34 44 PM" src="https://user-images.githubusercontent.com/35645434/99889068-29a8ec00-2c0f-11eb-8d05-1dc1483afebd.png">

Reply:
Looks great! Changing the toggle security to a link look makes both their intentions clear.